modules/sandboxed-containers-rhcos-extensions.adoc

```diff
@@ -0,0 +1,9 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+
+[id="sandboxed-containers-rhcos-extensions_{context}"]
+
+= {op-system} extensions
+
+The {sandboxed-containers-operator} is based on the {op-system-first} extensions concept. Kata and QEMU RPMs are installed as part of the sandboxed containers {op-system} extension. The extension contains the RPMs for kata, qemu-kiwi and its dependencies. You can enable them by using the `MachineConfig` resources that the Machine Config Operator provides.
```

Comment:
@jensfr I reworded a bit of what you wrote in your last comment. Does this look okay?
@zanetworker Please ack also.

Reply:
@sagidlow looks good to me, thanks! just a nit, make kata -> Kata, and qemu-kiwi -> QEMU

Reply:
@jensfr , I just realized there is two sentences repeating the same thing >.< 

*{op-system} - RHCOS
*{op-system-first} is Red Hat Core OS 

Updating to read as follows:

The {sandboxed-containers-operator} is based on the {op-system-first} extensions concept. The sandboxed containers {op-system} extension contains RPMs for Kata, QEMU, and its dependencies. You can enable them by using the `MachineConfig` resources that the Machine Config Operator provides.